index.css

```diff
@@ -0,0 +1,66 @@
+body {
+  background: none;
+}
+
+#list-cr {
+  border: 3px solid black;
+}
+
+.book-items {
+  display: flex;
+  justify-content: space-between;
+  background-color: #d3d3d3;
+}
+
+ul {
+  list-style: none;
+}
+
+hr {
+  width: 200px;
+}
+
+.append {
+  margin-top: 40px;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#add {
+  margin-left: 100px;
+}
+
+#form-head {
+  justify-content: center;
+  text-align: center;
+}
+
+#heading {
+  text-align: center;
+}
+
+#title {
+  margin-bottom: 15px;
+}
+
+#author {
+  margin-bottom: 15px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Kindly give styles to your pages your end result should look similar to this: 👇🏽 

| **Original design** | **Student design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/193413169-4a68c0b7-c5b8-42ab-b6b2-d4612fdb2638.png) | ![image](https://user-images.githubusercontent.com/58642949/193413201-74385779-2fa5-4070-910b-e9c3421788c1.png) |
| ![image](https://user-images.githubusercontent.com/58642949/193413241-2e220205-fee5-47c8-a2b2-c20ca68d0e3f.png) | ![image](https://user-images.githubusercontent.com/58642949/193413246-e8d48460-0938-4d50-baf9-17ec1cc50b31.png) |
| ![image](https://user-images.githubusercontent.com/58642949/193413293-4b0f1088-c262-4b71-b65c-bc4aeb84bd7d.png) | ![image](https://user-images.githubusercontent.com/58642949/193413268-26d9e267-d20a-4233-828c-d9e89e16a395.png) |